src/authentication/json-api-authorization-spec.ts

```diff
@@ -183,6 +187,49 @@ describe('authorization for api routes', () => {
         expect(api.postProjectHandler).to.not.have.been.called;
       });
     });
+    describe('getDeploymentCommentsHandler', () => {
+      it('should allow fetching comments for a deployment in an authorized project', async () => {
+        // Arrange
+        const { server, authentication, api } = await arrange(
+          'userHasAccessToDeployment',
+          true,
+          'getDeploymentCommentsHandler',
+        );
+        // Act
+        await makeRequest(server, '/comments/deployment/1-1', 'GET');
+
+        // Assert
+        expect(authentication.userHasAccessToDeployment).to.have.been.calledOnce;
```

Comment:
Should the response content and/or code be checked? Same thing for the test below.